docker-compose.yml

```
@@ -15,10 +15,10 @@ services:
     build:
       context: .
       args:
-        DATABASE_URL: postgresql://${POSTGRE_USERNAME}:${POSTGRE_PASSWORD}@localhost:5432/${POSTGRE_DATABASE}?schema=public
+        DATABASE_URL: ${DATABASE_URL}
```

There is a mistake there which is: `@localhost` should be replaced by `@db` since it's an internal network connection. However we don't need to add an env for that. `app` depends on `db`, this value can't be anything else.

```suggestion
        DATABASE_URL: postgresql://${POSTGRE_USERNAME}:${POSTGRE_PASSWORD}@db:5432/${POSTGRE_DATABASE}?schema=public
```

---

Yes, I was a bit uncomfortable with the duplicate information we have around this `DATABASE_URL` env variable, so it was a try to reuse i a little more :)

But we probably could act somewhere else :)